README.md

```diff
@@ -24,10 +30,22 @@ Please [email me](mailto:justin@shakacode.com) for a free half-hour project cons
 
 For more testimonials, see [Live Projects](PROJECTS.md) and [Kudos](./KUDOS.md).
 
+#### Features
+
+Like the [react-rails](https://github.com/reactjs/react-rails) gem, React on Rails is capable of server-side rendering with fragment caching and is compatible with [turbolinks](https://github.com/turbolinks/turbolinks). While the initial setup is slightly more involved, it allows for advanced functionality such as:
+
+- [Redux](https://github.com/reactjs/redux)
+- [Webpack optimization functionality](https://github.com/webpack/docs/wiki/optimization)
+- [React Router](https://github.com/reactjs/react-router)
+
+See the [react-webpack-rails-tutorial](https://github.com/shakacode/react-webpack-rails-tutorial) for an example of a live implementation and code.
 
 ----
+#### How it Works
```

Comment:
We should add a [blank line before the header](https://github.com/markdownlint/markdownlint/blob/e3dbc768e1608da5baab0a5896c54e840ff59110/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines).